src/_style.scss

```diff
@@ -0,0 +1,127 @@
+// Select
+// ===
+
+
+// Dependencies
+// ---
+
+@import 'vellum/dist/vellum';
+@import 'spline/dist/utilities/triangle';
+
+
+// Configurable Variables
+// ---
+//
+// 1. The same as default side paddings (but they're set in 1 variable)
+// 2. ($select__height - 20)/2 calculates vertical paddings
+
+$select__height: if(variable-exists(forms__element-height), $forms__element-height, 40px) !default;
+$select__padding: if(variable-exists(forms__element-padding), $forms__element-padding, ($select__height - 20)/2 50px ($select__height - 20)/2 15px) !default; // 2
+
+$select__border-color: if(variable-exists(border-color), $border-color, hsl(0, 0%, 10%)) !default;
+$select__border: if(variable-exists(border), $border, 1px solid $select__border-color) !default;
+$select__border-radius: if(variable-exists(forms__border-radius), $forms__border-radius, 0) !default;
+
+$select__background-color: if(variable-exists(forms__background-color), $forms__background-color, #fff) !default;
+$select__triangle-size: 7px !default;
+$select__triangle-position: 15px !default; // 1
+
+$select__font-family: if(variable-exists(forms__font-family), $forms__font-family, inherit) !default;
+$select__font-size: if(variable-exists(forms__font-size), $forms__font-size, inherit) !default;
+
+
+// Select Root
+// ---
+//
+// 1. Absolutely position select element and triangle relative to the parent .c-select
+
+.c-select {
+    position: relative; // 1
+
+    display: flex;
+    box-sizing: border-box;
+    max-width: 100%;
+    height: $select__height;
+    padding: $select__padding;
+    border: $select__border;
+
+    border-radius: $select__border-radius;
+    background: $select__background-color;
+
+    font-family: $select__font-family;
+    font-size: $select__font-size;
+
+    -webkit-user-select: none;
+
+    user-select: none;
+
+
+    // "Down" Triangle
+    // ---
+    //
+    // 1. Relative to the parent .c-select
+    // 2. Vertical center the triangle
+
+    &::after {
+        @include triangle(
+            $type: 'triangle',
+            $direction: 'down',
+            $size: $select__triangle-size,
+            $color: $select__border-color
+        );
+
+        content: '';
+
+        position: absolute; // 1, 2
+        top: 0;
+        right: $select__triangle-position;
+        bottom: 0;
+
+        margin: auto; // 2
+    }
+
+
+    // Select
+    // ---
+    //
+    // 1. Relative to the parent .c-select
```

Comment:
I like seeing comments like this as a general practice. Helps ensure there is no ambiguity and clarifies the purpose of stand alone properties when related properties are far out of site.